elado/yacht_kikoto.index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: elado
+title: Új tulajdonosra vár a színvonalas, népszerű, balatoni yacht kikötő!
+---
+
+#![Kívülről](http://i.imgur.com/4232j52.jpg) Új tulajdonosra vár a színvonalas, népszerű, balatoni yacht kikötő!
```

Comment:
> Jól vagytok? Félix élvezte az esküvőt? :)

Még kicsit náthásak vagyunk, úgyhogy itthon maradtam, és Bea is csak a szertartásra ment.